src/Codegen/Generator.hack

```diff
@@ -50,7 +50,7 @@ abstract class BaseObject<T as Field> implements GeneratableObjectType {
                 },
             )
             ->addDefault()
-            ->addLine("throw new \Error('Unknown field: '.\$field_name);")
+            ->addLine("throw new \Exception('Unknown field: '.\$field_name);")
```

Comment:
We can't catch an error using `expect(...)->toThrow()`, but we can catch an exception.